lesson07/assignment/src/views/KnowYourEgg.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+<div class="row pb-5">
+  <h1 class="col-12 pt-5 pb-2">Know Your Egg</h1>
+  <p class="lead col-12 pb-5">What do all those carton labels really mean? Click on each term for an explanation.</p>
+  <!-- Adding accordion -->
+  <div id="accordion" class="col-lg-10 col-md-10 col-sm-12">
+
+    <div class="card">
+      <div class="card-header pt-0 pb-0" id="termOne">
+        <h3 class="m-0 p-0">
+          <button class="btn" data-toggle="collapse" data-target="#collapseOne" aria-expanded="false" aria-controls="collapseOne"
+          v-for="term in terms" v-bind:key="term">{{term.termOne}}</button>
```

Comment:
You are very close on this one. You will want all of your accordion items to be in their own objects within an array. For example, some structure like this:
`terms = [ {
  "title": ""
  "text":""
},{
  "title": ""
  "text":""
},{
  "title": ""
  "text":""
}
]`

Then you would have one for loop for all of your accordions. It would loop through this array of objects, and populate your accordion. So you'd have "term.title" for your title, and the body text, something like "term.text".

Also, keys must be strings or integers, so you'd also want to set your key to be "term.title" or something else that would be unique per each object. Another alternative would be adding an ID field to each object above.

Reply:
Hi, Tyler. Being in the office hours session last week helped me figure this out (mostly). I was able to get the accordion to loop through all of my objects. The only issue now is that they all open and collapse together. :(